scripts/bundlingUtils.js

```diff
@@ -0,0 +1,135 @@
+const fs = require('fs');
+const path = require('path');
+
+// Function to update package.json
```

Comment:
could this comment be updated to why we need to update the pjson?  - same for all similar jsdoc-style comments